src/main/java/ch/hearc/cafheg/business/allocations/AllocationService.java

```diff
@@ -119,23 +122,30 @@ private String compareSalaryAccordingToIndependence(ParentAllocRequest r) {
      *                                           after initial checks have passed.
      */
     public void deleteAllocataire(long allocataireId) {
-        System.out.println("Service: Tentative de suppression de l'allocataire avec ID: " + allocataireId);
+        logger.info("Service: Tentative de suppression de l'allocataire avec ID: " + allocataireId);
 
         Allocataire allocataire = allocataireMapper.findById(allocataireId);
         if (allocataire == null) {
+            logger.error("Allocataire non trouvé avec ID: " + allocataireId);
```

Comment:
This is not a critical error, more an informational exception, `warn` level should be used instead of `error`
```suggestion
            logger.warn("Allocataire non trouvé avec ID: " + allocataireId);
```